.infra/kube/ingress-tls.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: tusd
+  namespace: tus
+  annotations:
+    kubernetes.io/tls-acme: "true"
+    kubernetes.io/ingress.class: "nginx"
+spec:
+  tls:
+  - hosts:
+    - dzybha.com
+    secretName: tusd-tls
+  rules:
+  - host: dzybha.com
```

Comment:
Still a leftover from experimenting?

Reply:
We actually deploy to this domain so we don't risk ruining anything. Once we are sure and want to merge to master we change this to our master domain. Otherwise we have no way of seeing the changes.